docs/account-management/plans.md

```diff
@@ -33,7 +33,12 @@ out to
 [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com)
 (please include which type of machine you prefer).
 
-**Note:** In Semaphore 2.0, all organizations are on monthly plans.
+In Semaphore 2.0, all organizations are on monthly plans.
+
+**Note:** If you are building open source projects on a Paid plan, those builds 
+will be charged accourding to our [pricing](https://semaphoreci.com/pricing).
```

Comment:
"If you are building open source projects on a Paid plan, the charges defined on our [pricing](https://semaphoreci.com/pricing) page will apply." **or** "If you are building open source projects on a Paid plan, you will be charged according to our [pricing](https://semaphoreci.com/pricing) page."

pls note that you have a typo in 'according' (accourding)